tools/java/org/unicode/cldr/util/UnitPreferences.java

```diff
@@ -56,7 +56,7 @@ public String toString() {
     }
 
     static final Splitter SPLIT_SPACE = Splitter.on(' ').trimResults().omitEmptyStrings();
-    static final Joiner JOIN_COLON = Joiner.on(':');
+    static final Joiner JOIN_AND = Joiner.on("-and");
```

Comment:
Missing a `-` ?

Reply:
Good catch; the tests are not updated to check for that.

Reply:
Will fix.

Reply:
Fixed it; turns out that it was dead code; found another piece of dead code also.